Steering Group/proto-Steering Group/Minutes/20241210.md

```diff
@@ -0,0 +1,79 @@
+# 2024-12-10 Proto-steering group meeting
+
+Present:
+- Birgit Hassler (chair)
+- Alistair Sellar
+- Axel Lauer
+- Bouwe Andela (notes)
+- Ranjini Swaminathan
+- Julien Lenhardt
+- Saskia Loosveldt Tomas
+
+
+## Agenda
+
+1. Standing items:
+    * Actions from previous meetings
+    * Update on funding proposals
+    * Updates from (ESMValTool and wider) community discussions
+    * Next workshop
+    * Consortium agreement
+1. Dedicated note taker
+1. Monthly community meetings
+1. Governance checklist: https://sustainers.github.io/governance-readiness/ (Bouwe to introduce)
+
```

Comment:
Should the numbering not go from 1. to 4.?

Reply:
That is taken care of automatically: https://github.com/ESMValGroup/Community/blob/steering-group-20241210/Steering%20Group/proto-Steering%20Group/Minutes/20241210.md#agenda

Reply:
Ahh, I see! All good then.  :)